src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx

```diff
@@ -273,14 +279,16 @@ export const AccuracyCorrectionAndRetention = () => {
                 </p>
               )}
             </div>
-            <ViewComments
-              count={
-                commentCount?.[
-                  PiaSections.ACCURACY_CORRECTION_AND_RETENTION_ACCURACY
-                ]
-              }
-              path={PiaSections.ACCURACY_CORRECTION_AND_RETENTION_ACCURACY}
-            />
+            {!hideViewComments && (
```

Comment:
would it be better to have it default to view comment so we avoid the ( ! ) NOT operator.

Reply:
Checking for positive outcome is more readable.

`!hideViewComments` >>> `showComments`